downstream/assemblies/devtools/assembly-rhdh-using.adoc

```diff
@@ -11,6 +11,13 @@ From the {RHDH} UI, you can navigate to your {PlatformNameShort} instance, where
 This document describes how to use the {AAPRHDH}.
 It presents a worked example of developing a playbook project for automating updates to your firewall configuration on RHEL systems.
 
+== Optional requirement
+
+The {AAPRHDH} link to Learning Paths on the Red Hat developer portal,
```

Comment:
I don't know how strict the "Red{nbsp}Hat" rule, but the style guide does state that any instance of Red Hat should have a non breaking space so it doesn't run over two lines and separate due to formatting. I'll let you decide if you want to follow it or not as it's kind of a silly rule.